### DISCUSSION TITLE
Pls add thermostat[wk] support

### DISCUSSION THREAD

**Discussion by rextsou0623** [tuya/tuya-homebridge#127] [Ideas]
Tuya developers, thanks for your awesome work on getting this homebridge plugin. Can you please add thermostat support? I don't see any reason why it's excluded because so many ESP thermostats out there. Is it out of security concern?
Thanks!

**Comment by ATomekGH** [comment]
is there any chance to add support for the HY368RT Zigbee thermostat?

**Comment by rextsou0623** [comment]
I live in Mainland China. My air conditioner supplier (YORK) installed their thermostat. It can connect to wifi, and controlled by Tuya app on iOS. I wish I could link them to homekit via bridge. Thanks.

**Comment by istationonline** [comment]
Please add thermostat support, why is this not available?
